studio/pages/sign-in-mfa.tsx

```diff
@@ -31,8 +32,9 @@ const SignInMfaPage: NextPageWithLayout = () => {
           const { data, error } = await auth.mfa.getAuthenticatorAssuranceLevel()
           if (error) {
             // if there was a problem signing in via the url, don't redirect
+            toast.error(`Failed to retrieve assurance level: ${error.message}`)
```

Comment:
Worth adding some action for the user to take here? Like:
```suggestion
            toast.error(`Failed to retrieve assurance level: ${error.message}. Please try signing in again`)
```

Reply:
lol @kangmingtay already beat me to it